### DISCUSSION TITLE
Combobox is missing from documentation

### DISCUSSION THREAD

**Discussion by ACPK** [tailwindlabs/headlessui#1093] [Help]
I'm able to see the Combobox component on Github, but I am not seeing it on https://headlessui.dev.

Here is the code I used: https://github.com/tailwindlabs/headlessui/tree/fbaa1ae9da8788910ce8b240851f9779630babcb/packages/playground-react/pages/combobox

    let people = [
      { value: 'alice', name: 'alice' },
      { value: 'bob', name: 'bob' },
      { value: 'charlie', name: 'charlie' },
    ]

    function Example(props) {
      let [value, setValue] = useState(undefined)
      let [query, setQuery] = useState('')

      let filteredPeople =
        query === ''
          ? people
          : people.filter(person => person.name.toLowerCase().includes(query.toLowerCase()))

      return (
        <Combobox value={value} onChange={setValue}>
          <Combobox.Input onChange={event => setQuery(event.target.value)} />
          <Combobox.Button>↕</Combobox.Button>
          <Combobox.Options>
            {filteredPeople.map(person => (
              <Combobox.Option key={person.value} value={person.value}>
                {person.name}
              </Combobox.Option>
            ))}
          </Combobox.Options>
        </Combobox>
      )
    }

**Comment by recurrence** [comment]
I believe it's not actually released yet.

**Reply by RobinMalfait** [reply]
Correct, the Combobox is still in development and not released yet. We are working on the docs as we speak!

**Comment by recurrence** [comment]
It works great by the way (I'm using it from the main branch) :)
